src/garage/torch/algos/ppo.py

```diff
@@ -42,16 +49,18 @@ class PPO(VPG):
             the mean entropy to the surrogate objective. See
             https://arxiv.org/abs/1805.00909 for more details.
         minibatch_size (int): Batch size for optimization.
-        max_optimization_epochs (int): Maximum number of epochs for update.
+        optimization_epochs (int): Number of epochs for update.
```

Comment:
please continue to use max_optimization_epochs. there's no reason this algorithm can't use early stopping in the future.